scripts/.flake8

```diff
@@ -29,10 +29,10 @@ per-file-ignores =
     r.in.srtm/r.in.srtm.py: E722
     r.fillnulls/r.fillnulls.py: E722
     d.rast.edit/d.rast.edit.py: E722
-    g.manual/g.manual.py: E722
-    v.to.lines/v.to.lines.py: E722
-    v.in.wfs/v.in.wfs.py: E722
-    r.plane/r.plane.py: E722
+    
+    
+    
+    
```

Comment:
Just delete the lines. No need to keep them.

Reply:
Now I have deleted these 4 lines from scripts/.flake8. Kindly review.